unison-src/transcripts/child-namespace-history-merge.md

```diff
@@ -0,0 +1,95 @@
+# Behaviour of namespace histories during a merge.
+
+Note: This is a descriptive test meant to capture the current behaviour of 
+branch histories during a merge.
+It isn't prescriptive about how merges _should_ work with respect to child branches, 
+but I think we should at least notice if we change things by accident.
+
+
+## Setting up some history
+
+```ucm:hide
+.> builtins.merge
+```
+
+```unison
```

Comment:
```suggestion
```unison:hide
```

This will hide the type-checking output, which is basically echoed verbatim in the `add` step.  Don't hide it if you want to show something about the type-checking error though, or don't plan to `add` immediately after and want to see the types printed.